mockup_package/mockup/image_generator.py

```diff
@@ -43,17 +43,20 @@ def create_fit_resolution_image(self):
         # some picture has orientation property, which the browser knows and can automatically rotate the image # noqa: E501
         # but the property is lost when we mockup the image, so need to fix it manually. # noqa: E501
         # ref: https://stackoverflow.com/questions/13872331/rotating-an-image-with-orientation-specified-in-exif-using-python-without-pil-in # noqa: E501
-        for orientation in ExifTags.TAGS.keys():
-            if ExifTags.TAGS[orientation] == "Orientation":
-                break
-
-        exif = image.getexif()
-        if exif[orientation] == 3:
-            image = image.rotate(180, expand=True)
-        elif exif[orientation] == 6:
-            image = image.rotate(270, expand=True)
-        elif exif[orientation] == 8:
-            image = image.rotate(90, expand=True)
+        try:
+            for orientation in ExifTags.TAGS.keys():
+                if ExifTags.TAGS[orientation] == "Orientation":
+                    break
+
+            exif = image.getexif()
+            if exif[orientation] == 3:
+                image = image.rotate(180, expand=True)
+            elif exif[orientation] == 6:
+                image = image.rotate(270, expand=True)
+            elif exif[orientation] == 8:
+                image = image.rotate(90, expand=True)
+        except:
+            pass
```

Comment:
```suggestion
          except SpecificOrientationException:
            # image does not have orientation, expected case, will silent error
            pass
```

Reply:
@pkong-ds added comment, but I'm just using `Exception` as not sure what exception it sends

Reply:
@YayunHuang I added a commit for this. Can help review if ok?

Reply:
https://stackoverflow.com/a/9824050/19287186